pkg/composer/up_service.go

```diff
@@ -43,6 +43,10 @@ func (c *Composer) upServices(ctx context.Context, parsedServices []*servicepars
 		}
 	}
 
+	if uo.NoStart {
+		return nil
```

Comment:
This should call `nerdctl create` (which isn't implemented yet)

Reply:
`nerdctl compose create` ? `docker-compose create` has been deprecated. 
https://docs.docker.com/compose/reference/create/

Reply:
No, `nerdctl create` (`docker create`)  to create non-running containers